spec/rspec/core/lookup_set_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe 'RSpec::Core::LookupSet' do
+
+  let(:set) { RSpec::Core::LookupSet.new([1, 2, 3]) }
+
+  it 'takes an array of values' do
+    expect(set).to include(1).and include(2).and include(3)
```

Comment:
This can just be `expect(set).to include(1, 2, 3)` -- no need to make it a compound.

Also, the expectation doesn't seem connected to the doc string -- the expectation seems to show that it supports inclusion checks.  The `let(:set)` shows that it takes an array of values, but all the examples show that since they all call `set`.